napari/utils/perf/_compat.py

```diff
@@ -3,11 +3,14 @@
 import sys
 import time
 
-if sys.version_info[:2] >= (3, 7):
-    # Use the real perf_counter_ns
-    perf_counter_ns = time.perf_counter_ns
-else:
-
-    def perf_counter_ns():
-        """Compatibility version for pre Python 3.7."""
-        return int(time.perf_counter() * 1e9)
+GREATER_EQUAL_PY37 = sys.version_info[:2] >= (3, 7)
```

Comment:
```
setup.cfg
38:python_requires = >=3.7
```

So this is likely always true. Also on Python 3+ you don't need the `[:2]`.

Might be for a separate PR.

Reply:
Noted on #2599